client/analytics/report/products/table.js

```diff
@@ -0,0 +1,227 @@
+				isLeftAligned: true,
+			},
+			{
+				label: __( 'SKU', 'wc-admin' ),
+				key: 'sku',
+				hiddenByDefault: true,
+			},
+			{
+				label: __( 'Items Sold', 'wc-admin' ),
+				key: 'items_sold',
+				required: true,
+				defaultSort: true,
+				isSortable: true,
+				isNumeric: true,
+			},
+			{
+				label: __( 'G. Revenue', 'wc-admin' ),
+				key: 'gross_revenue',
+				required: true,
+				isSortable: true,
+				isNumeric: true,
+			},
+			{
+				label: __( 'Orders', 'wc-admin' ),
+				key: 'orders_count',
+				isSortable: true,
+				isNumeric: true,
+			},
+			{
+				label: __( 'Category', 'wc-admin' ),
+				key: 'product_cat',
+			},
+			{
+				label: __( 'Variations', 'wc-admin' ),
+				key: 'variation',
+			},
+			{
+				label: __( 'Status', 'wc-admin' ),
+				key: 'stock_status',
+			},
+			{
+				label: __( 'Stock', 'wc-admin' ),
+				key: 'stock',
+			},
+		];
+	}
+
+	getRowsContent( data = [] ) {
+		const { stockStatuses } = wcSettings;
+
+		return map( data, row => {
+			const {
+				product_id,
+				sku = '', // @TODO
+				name,
+				items_sold,
+				gross_revenue,
+				orders_count,
+				categories = [], // @TODO
+				variations = [], // @TODO
+				stock_status = 'outofstock', // @TODO
+				stock_quantity = '0', // @TODO
+			} = row;
+
+			return [
+				{
+					// @TODO Must link to the product detail report.
+					display: name,
+					value: name,
+				},
+				{
+					display: sku,
+					value: sku,
+				},
+				{
+					display: items_sold,
+					value: items_sold,
+				},
+				{
+					display: formatCurrency( gross_revenue ),
+					value: getCurrencyFormatDecimal( gross_revenue ),
+				},
+				{
+					display: (
+						<Link
+							href={
+								'admin.php?page=wc-admin#/analytics/orders?filter=advanced&product_includes=' +
+								product_id
+							}
+							type="wp-admin"
+						>
+							{ orders_count }
+						</Link>
+					),
+					value: orders_count,
+				},
+				{
+					display: categories.map( cat => cat.name ).join( ', ' ),
+					value: categories.map( cat => cat.name ).join( ', ' ),
+				},
+				{
+					display: variations.length,
+					value: variations.length,
+				},
+				{
+					display: (
+						<Link href={ 'post.php?action=edit&post=' + product_id } type="wp-admin">
+							{ stockStatuses[ stock_status ] }
+						</Link>
+					),
+					value: stockStatuses[ stock_status ],
+				},
+				{
+					display: stock_quantity,
+					value: stock_quantity,
+				},
+			];
+		} );
+	}
+
+	renderPlaceholderTable( tableQuery ) {
+		const headers = this.getHeadersContent();
+
+		return (
+			<Card
+				title={ __( 'Products', 'wc-admin' ) }
+				className="woocommerce-analytics__table-placeholder"
+			>
+				<TablePlaceholder
+					caption={ __( 'Products', 'wc-admin' ) }
+					headers={ headers }
+					query={ tableQuery }
+				/>
+			</Card>
+		);
+	}
+
+	renderTable( tableQuery ) {
+		const { products, query, totalRows } = this.props;
+
+		const rowsPerPage = parseInt( tableQuery.per_page ) || 25;
```

Comment:
Wondering if we should make a global QUERY_DEFAULTS object of sorts so we have one place to update the default page size across the app. Could do this as a follow-up, but would also potentially allow for this to be a user-based setting in the future.

Reply:
Default date presets could go here as well

Reply:
That sounds really good. I created an issue to track that: #634.